elasticdl/proto/elasticdl.proto

```diff
@@ -112,7 +112,8 @@ message ReportEvaluationMetricsResponse {
 message ReportEvaluationMetricsRequest {
     // Model version used to compute evaluation metrics.
     int32 model_version = 1;
-    map<string, Tensor> evaluation_metrics = 2;
+    map<string, Tensor> model_outputs = 2;
+    Tensor labels = 3;
```

Comment:
can we only use a `map` instead of a `map` plus `labels`?

Reply:
The keys of `model_outputs` are defined by users. It seems difficult to put `labels` into that map?

Reply:
By making the reporting data more general, related logic can also be more flexible.

Reply:
It is difficult to make reporting data more general. We can modify the proto file in another PR if we have a better way.